examples/item2.json

```diff
@@ -21,6 +21,7 @@
       "msi"
     ],
     "eo:cloud_cover": 51.580326,
+    "eo:snow_cover": 0,
```

Comment:
one thing to consider is if we want to have Yet Another Extension (if we don't already have one, there are so many!) for scene analytics, and we move the eo cloud_cover and snow_cover fields there, along with no data percentage and other descriptions of the scene. But for this change, this is the most reasonable approach right now.